### ISSUE TITLE
Cannot compile module with android 23 API :  BookmarkColumns is not found

### ISSUE THREAD

**Issue by kio21** [appcelerator-archive/ti.barcode#35]
I got some errors connected with not found `BookmarkColumns` symbol:

```
[javac] /Users/kio21/git/ti.barcode/android/src/com/google/zxing/client/android/share/AppPickerActivity.java:49: error: cannot find symbol
[javac]       intent.putExtra(Browser.BookmarkColumns.URL, url);
[javac]                              ^
[javac]   symbol:   variable BookmarkColumns
[javac]   location: class Browser
```

**Comment by AppWerft** [comment]
+1 same issue

**Comment by markruys** [comment]
Think this is the same issue:
```
$ ti build -p android --build-only
...
[ERROR] Failed to compile Java source files:
[ERROR]   
[ERROR] warning: [options] bootstrap class path not set in conjunction with -source 1.6
[ERROR] Note: [KrollBindingGen] Running Kroll binding generator.
[ERROR] Note: [KrollBindingGen] No binding data found, creating new data file: org.appcelerator.titanium.bindings/barcode.json
[ERROR] Note: [KrollBindingGen] Found binding for module Barcode
[ERROR] Note: [KrollBindingGen] Generating JSON: file:/Users/markr/Downloads/naamlozemap/ti.barcode/android/build/generated/json/org/appcelerator/titanium/bindings/barcode.json
[ERROR] warning: The following options were not recognized by any processor: '[kroll.checkTiContext]'
[ERROR] /Users/markr/Downloads/naamlozemap/ti.barcode/android/src/com/google/zxing/client/android/share/AppPickerActivity.java:49: error: cannot find symbol
[ERROR]       intent.putExtra(Browser.BookmarkColumns.URL, url);
[ERROR]                              ^
[ERROR]   symbol:   variable BookmarkColumns
[ERROR]   location: class Browser
```

**Comment by markruys** [comment]
I think it's related to [Android - Browser.BookmarkColumns cannot be resolved in Zxing Library](http://stackoverflow.com/questions/33996683/android-browser-bookmarkcolumns-cannot-be-resolved-in-zxing-library). The Titanium SDK that I use, 5.5.1, needs Android API 23. When I temporary hack mobilesdk/osx/5.5.1.GA/android/package.json and set vendorDependencies.androidsdk to 22, the module compiles indeed. 

Then I tried the last API 22 SDK: 5.0.1. To my dismay compiling gave me another issue:
~~~
Error: Cannot find module '/Users/markr/Library/Application Support/Titanium/mobilesdk/osx/5.0.1.GA/node_modules/sqlite3/lib/binding/node-v46-darwin-x64/node_sqlite3.node'
~~~
node-v44 does exists. Perhaps this is an issue of TiShadow. Pfff.

**Comment by hansemannn** [comment]
Resolved in the latest 3.0.0 release
